src/anw-customselect.html

```diff
@@ -164,8 +156,34 @@
                 this.shadowRoot.querySelector('#datePicker').value = "";
             }
 
+            _locate(e) {
+                var spinner = this.shadowRoot.querySelector('#locationSpinner');
+                if (navigator.geolocation) {
+                    spinner.active = true;
+                    navigator.geolocation.getCurrentPosition((position) => {
+                        {
+                            const locationAPIURL = `https://nominatim.openstreetmap.org/reverse?` +
```

Comment:
Vorschlag: 
- Niederlassungen (CampusE, IT-Rathaus, TenTowers) definieren und im gewissen Radius diese nehmen
- Openstreetmap als Fallback